### PR TITLE
CUMULUS-350 Update queue-granules to use cumulus-message-adapter

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

**Comment by yjpa7145** [comment]
In addition to updating the queue-granules task to use the message adapter, this PR also greatly simplifies the granule queuing code.

File: cumulus/tasks/queue-granules/index.js [hunk 1]
```diff
@@ -1,52 +1,42 @@
...
-* @param  {object} event lambda event object
-* @param  {object} event.input
-* @param  {array} event.input.granules
-* @param  {object} context Lambda context object. See https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html
-* @param  {lambdaCallback} callback callback function
-* @return {undefined}
+* See schemas/input.json and schemas/config.json for detailed event description
+*
+* @param {Object} event - Lambda event object
+* @returns {Promise} - see schemas/output.json for detailed output schema
```
Comment (laurenfrederick) [inline cumulus/tasks/queue-granules/index.js]: This is returning an Object, not a Promise, right?

File: cumulus/tasks/queue-granules/package.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@cumulus/common": "^1.0.1",
+    "@cumulus/cumulus-message-adapter-js": "0.0.1-beta.3",
```
Comment (laurenfrederick) [inline cumulus/tasks/queue-granules/package.json]: We should make this 1.0.0

File: cumulus/tasks/queue-granules/index.js (hunk 1 above)
Comment (yjpa7145) [reply cumulus/tasks/queue-granules/index.js]: It's an async function, so it's returning a Promise.

File: cumulus/tasks/queue-granules/package.json (hunk 2 above)
Comment (yjpa7145) [reply cumulus/tasks/queue-granules/package.json]: I created CUMULUS-367 to deal with this.

**Pull request merged**